cisagov/XFD
# Resolves issue where inventory ui state stagnates and fails to render as expected

#  #

## 🗣 Description ##




- Updated DashboardUI component to setSearchTerm on unmount 
- Prevent state stagnation where the UI would not correctly render after refreshing the view

## 💭 Motivation and context ##






Resolves https://maestro.dhs.gov/jira/browse/CRASM-325

## 🧪 Testing ##





- Navigate to inventory dashboard. 
- Click "Search Results"
- Refresh the page (regardless of refreshing, the UI will render as expected)



## ✅ Pre-approval checklist ##






      in code comments.
      to reflect the changes in this PR.

## ✅ Pre-merge checklist ##






## ✅ Post-merge checklist ##
